### DISCUSSION TITLE
CLI and API usage (question about future update)

### DISCUSSION THREAD

**Discussion by ultrominka** [4sval/FModel#434] [Q&A] [answered]
Can you tell me if there will be a future implementation of the FModel using CLI commands (like in UModel)? That would be very convenient!

**Comment by 4sval** [comment, marked as answer]
https://github.com/FabianFG/CUE4Parse

**Comment by joric** [comment]
I've made a command-line tool https://github.com/joric/CUE4Parse.CLI/wiki
